### PR TITLE
Fastly request

### PR DESCRIPTION
**Pull request by cmijaresb**
Through #cloud-fastly requested to update change geoip.country_code to client.geo.country_code
https://experiencecloud.slack.com/archives/CGPNB2YE8/p1625248197390000

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@cmijaresb thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by hguthrie** [comment]
Hi there. Thanks for taking the time to update that.
After you sign the Adobe CLA ☝️, we can process your change.

File: src/cloud/cdn/fastly-vcl-blocking.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ This example uses the two-character ISO 3166-1 country code for the country asso
   "dynamic": "0",
   "type": "recv",
   "priority": "5",
-  "content": "if ( geoip.country_code == \"HK\" ) { error 405 \"Not allowed\";}"
+  "content": "if ( client.country_code == \"HK\" ) { error 405 \"Not allowed\";}"
```
Comment (smaeda-ks) [inline src/cloud/cdn/fastly-vcl-blocking.md:121]:
```suggestion
  "content": "if ( client.geo.country_code == \"HK\" ) { error 405 \"Not allowed\";}"
```

Comment (smaeda-ks) [reply src/cloud/cdn/fastly-vcl-blocking.md:121]: @cmijaresb ^^

**Comment by dobooth** [comment]
Closing for lack of CLA.

**Comment by smaeda-ks** [comment]
301 to https://github.com/magento/devdocs/pull/9055

**Pull request closed without merging**